cumulus_library_opioid/vocab/static_builder.py

```diff
@@ -30,27 +30,34 @@ class StaticBuilder(base_table_builder.BaseTableBuilder):
 
     tables = [  # noqa: RUF012
         TableConfig(
-            file_path=base_path / "./common/keywords/keywords.curated.tsv",
+            file_path=base_path / "./common/keywords/keywords.tsv",
             delimiter="\t",
             table_name="keywords",
             headers=["STR"],
             dtypes={"STR": "str"},
             parquet_types=["STRING"],
-            filtered_path=base_path / "./common/keywords/keywords.curated.filtered.tsv",
+            filtered_path=base_path / "./common/keywords/keywords.filtered.tsv",
+        ),
+        TableConfig(
+            file_path=base_path / "./all_rxcui_str.RXNCONSO_curated.tsv",
+            delimiter="\t",
+            table_name="all_rxnconso_keywords",
+            headers=["RXCUI","STR","TTY","SAB","CODE","keyword","keyword_len"],
+            dtypes={"RXCUI":"str","STR":"str","TTY":"str","SAB":"str","CODE":"str","keyword":"str","keyword_len":"str"},
+            parquet_types=["STRING","STRING","STRING","STRING","STRING","STRING","STRING"],
         ),
         TableConfig(
             file_path=base_path / "./common/expand_rules/expand_rules.tsv",
             delimiter="\t",
-            table_name="expand_rules",
-            # TODO: do we actually need to preserve 'rule'?
+            table_name="search_rules",
             headers=[
                 "TTY1",
                 "RELA",
                 "TTY2",
                 "rule",
             ],
             dtypes={"TTY1": "str", "RELA": "str", "TTY2": "str", "rule": "str"},
-            parquet_types=["STRING", "STRING", "STRING", "STRING", "STRING"],
+            parquet_types=["STRING", "STRING", "STRING", "STRING", "BOOLEAN"],
```

Comment:
What's happening here. Looks like there are four columns, all strings. But then there's a fifth bool column now?

Reply:
the source file has four columns. the map_cols structure below allows you to pivot one column to another, and map over a subset of values to the new column, generating that extra column.